packages/@aws-cdk/aws-logs/test/destination.test.ts

```diff
@@ -65,4 +65,24 @@ describe('destination', () => {
       TargetArn: 'arn:bogus',
     });
   });
+
+  test('fromDestinationArn', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
```

Comment:
This is a good test case, but I'd like to see two more:

1. Cross stack, within the same account
2. Cross stack, between two different accounts